pkgs/by-name/vs/vst-sdk/package.nix

```diff
@@ -0,0 +1,15 @@
+{
+  lib,
+  fetchzip,
+}:
+fetchzip rec {
+  name = "vst-sdk-${version}";  # cannot be `pname`, as `fetchzip` expects `name`
+  version = "2018-06-11";
```

Comment:
Maybe indicate this as `unstable`? Might be unnecessary though.

Reply:
Was this not a stable release?  I have no clue about that SDK, even though I know some about computer-assisted music... and want to get back to doing music in general, but that's not the right place for that conversation  :sweat_smile:

Reply:
I have no idea either honestly. Feel free to disregard this comment.

Reply:
Can you clarify what you mean by “doesn't like this” ?

Reply:
AFAIK, Stienberg wants to move everyone to VST3 nowadays, and its incredibly hard to get your hands on the original VST2 SDK. 

This is why we have to do the whole archive.org thing, because they tried remove the VST2 SDK from the internet at one point.

We're probably fine though.

Reply:
Ah, I see.  Yeah, it makes sense to keep it around for legacy reasons, then.

Reply:
Should we only include the `VST2_SDK` directory then, to ensure nobody accidentally uses an ancient version of the VST3 SDK?

Reply:
The VST3 SDK is freely available, so most if not everyone probably has it vendored.